package.json

```diff
@@ -28,8 +28,10 @@
     "url": "https://github.com/PolymerLabs/tattoo/issues"
   },
   "dependencies": {
+    "@types/colors": "^0.6.33",
     "bottleneck": "^1.10.2",
     "bower": "^1.7.7",
+    "colors": "^1.1.2",
```

Comment:
we've been using chalk everywhere else in our ecosystem, it would be good to standardize on that